rpc_util.go

```diff
@@ -41,6 +42,16 @@ import (
 	"google.golang.org/grpc/transport"
 )
 
+// These constants values are copied from the flate package, Declared to generify compression for all
```

Comment:
These also exist in the `gzip` package; I don't think we want to copy them here.